root/edit/components/EditSidebar.js

```diff
@@ -75,7 +75,7 @@ const EditSidebar = ({$c, edit}: Props) => (
         )}
       </SidebarProperty>
 
-      <SidebarProperty className="" label={l('Conditions:')}>
+      <SidebarProperty className="" label={l('If no votes cast:')}>
```

Comment:
Isn't "If not enough votes cast" more accurate to say?

Reply:
I guess. It's just long, but we can probably live with long.

Reply:
Wait, actually, is it? I thought getEditExpireAction only mattered if no votes at all were cast, right? (well, I guess nowadays it doesn't really matter anywhere, but for old edits it did).

Reply:
Checked the code and you're right. Technically it'll also perform the expire action if only abstain votes are cast, but that's basically "no votes" rather than "not enough." https://github.com/metabrainz/musicbrainz-server/blob/9fe476c5f6bc0ae4b80c5e532a99bce0dae7a069/lib/MusicBrainz/Server/EditQueue.pm#L211